horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/ServiceMediator.java

```diff
@@ -168,6 +167,18 @@ void queueRunRecalculation(int runId) {
         runEmitter.send(runId);
     }
 
+    @Incoming("schema-sync-in")
+    @Blocking(ordered = false, value = "horreum.run.pool")
```

Comment:
We will want to configure this pool size independent of the `horreum.run.pool`, We should introduce a new configuration option to size this consumer pool

Reply:
Yeah, you are right.

Reply:
Added `horreum.schema.pool` configuration with commit https://github.com/Hyperfoil/Horreum/pull/1581/commits/2a5aa2fa27d280b24018c9787b4a8ad27b2737de